src/color.scss

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2021 Alaska Airlines. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+/* stylelint-disable custom-property-empty-line-before, declaration-no-important */
+
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables';
+
+:host {
+  background-color: var(--ds-auro-menuoption-background-color);
+  color: var(--ds-auro-menuoption-text-color);
+}
+
+:host([disabled]) {
+  --ds-auro-menuoption-text-color: var(--ds-color-text-ui-disabled-default, $ds-color-text-ui-disabled-default);
+}
+
+:host(:hover:not([disabled])),
+:host(.active:not([disabled])) {
```

Comment:
I am not sure this is a valid selector given how the component works. If you look at the JS file you will see that the functionality that adds 'active' to the class list we first check that the menuOption is valid for that - and if it's disabled, it's not. So `.active' and `[disabled]` should never be on the same option at the same time.

Given that, I think your selector can just be `.active {...}`.